cli/Sources/TuistAcceptanceTesting/TuistAcceptanceFixtures.swift

```diff
@@ -36,6 +36,7 @@ public enum TuistAcceptanceFixtures {
     case commandLineToolWithStaticLibrary
     case frameworkWithEnvironmentVariables
     case frameworkWithMacroAndPluginPackages
+    case frameworkWithMacrosAndTests
```

Comment:
I'd not include it here since there's no value in doing so. I'd refer them using their name as a string instead.